test/test-server/backend.go

```diff
@@ -102,3 +102,11 @@ func (b *TestBackend) GetKafkaRestUrl() string {
 func (b *TestBackend) GetMdsUrl() string {
 	return b.mds.URL
 }
+
+func EnableMdsSso() {
+	isSsoEnabled = true
+}
+
+func DisableMdsSso() {
+	isSsoEnabled = false
+}
```

Comment:
Just make this variable public?

Reply:
I wonder if we could use this pattern elsewhere. We have a weird process for enabling/disabling audit logs

Reply:
Good point; I'm including the test-server package anyway to access these functions so I'll delete them and make the var public.

I don't see any reason why we couldn't do this for audit logs too.

Side note on audit logs: I initially tried to set this up so that it would close and restart the service with a different value for the sso switch like the current audit log pattern, but for some reason I couldn't figure out, the server from `httptest.NewServer` refuses any connections if it's closed and restarted. It's the only testing backend that isn't started with `newTestCloudServer`, and none of those seem to be affected.